demos/image-classification/index.js

```diff
@@ -181,6 +183,18 @@ const main = async () => {
     }
   } catch (err) {
     log(`[Error] ${err}`);
+
+    status.setAttribute("class", "red");
+    info.innerHTML = `
+            ${error}<br>
+            Your device probably doesn't have a supported GPU.`;
+    label_uploadImage.setAttribute("class", "disabled");
+    uploadImage.disabled = true;
+    classify.disabled = true;
+    log(`[Error] ${error}`);
+    log(
+      `[Error] Your device probably doesn't have a supported GPU`
+    );
```

Comment:
```suggestion
    log(`[Error] Your device probably doesn't have a supported GPU`);
```
Same line? (it's not even 80 columns wide, which are pretty dubious in 2024)